cpp/src/generic_players/main/UnitTests.cpp

```diff
@@ -15,6 +16,8 @@
 
 namespace generic {
 
+using tictactoe = game_transform::AddStateStorage<tictactoe::Game>;
```

Comment:
The clash with the `tictactoe` namespace can potentially lead to confusion. Please rename to `TicTacToe`.